java/tournament-scheduling/src/main/resources/application.properties

```diff
@@ -39,5 +39,4 @@ quarkus.swagger-ui.always-include=true
 # Test overrides
 ########################
 # Effectively disable spent-time termination in favor of the best-score-limit
-%test.quarkus.timefold.solver.termination.spent-limit=1h
-%test.quarkus.timefold.solver.termination.best-score-limit=0hard/*medium/*soft
\ No newline at end of file
+%test.quarkus.timefold.solver.termination.spent-limit=10s
```

Comment:
Is this change correct?

Reply:
Yes. Unfortunately, for `BigDecimal`, we cannot use `*`.
This is the next best thing IMO. Do you have a different idea?

Reply:
Right. Since we have a green build, `10s` looks good.